app/models/application_form.rb

```diff
@@ -8,6 +8,10 @@ class ApplicationForm < ApplicationRecord
   has_many :application_qualifications
   has_many :references
 
+  after_save -> {
+    application_choices.update_all(updated_at: Time.now)
```

Comment:
I think it's safer to use `Time.zone.now`. `Time.now` returns system time which is probably the same on our machines but might not be on a server (I think).